lib/scrabble.rb

```diff
@@ -15,4 +13,81 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+
+  def score(word)
+    if word == nil
+      return 0
+    elsif word.is_a? Integer
```

Comment:
I like that you're looking out for edge cases! This wouldn't quite catch all cases though (what if I passed in a symbol or a hash, etc.?). Think about ways you could check that it's not a string or that it only includes alphabet letters.